Documentation/gettingstarted/istio.rst

```diff
@@ -22,6 +22,11 @@ environment running on your machine.
    instructions provided here for the other GSGs. 5 GB and 4 CPUs
    should be enough for this GSG (``--memory=5120 --cpus=4``).
 
+.. note::
+
+   If Cilium is deployed in kube-proxy-free mode, you need to set
```

Comment:
We could add some more details here. At the very least, specify the command to add the option or clarify that it needs to be set in the cilium deployment yaml.

Reply:
Provided more details, PTAL.